### ISSUE TITLE
Package is not showing on android device

### ISSUE THREAD

**Issue by tonvlad88** [tongyy/react-native-draggable#30]
Hello, I'm new on react native. Can you help me with this android issue.. When I tried to implement this package into my app, it is showing on my Iphone 6 but not on my Huawei Nova 5t. Below is my code in my App.js: 

> App.js

![image](https://user-images.githubusercontent.com/40188798/67757123-2b3cca80-fa76-11e9-876c-8be9ae8d9e7d.png)

> IOS view

![IOS](https://user-images.githubusercontent.com/40188798/67756743-86ba8880-fa75-11e9-86e5-fac6bc38ff78.png)

> Android view

![Android](https://user-images.githubusercontent.com/40188798/67756744-87531f00-fa75-11e9-9050-6ffdf7126577.jpg)

It seems not working on my android but working on IOS. Can you advise if there something I missed on my code? I have tried adding z and zindex in style but still no luck.

**Comment by tongyy** [comment]
are there any errors in the console? if there are no errors at all, can you play around the style of QuickMenu such as z-index or position

**Comment by tonvlad88** [comment]
Hi @tongyy thanks for the fast response. I managed to show it now but unfortunately I cannot drag it on android but i can on IOS. Is there any property that it can be drag on android? i would love to use it just like the assistive touch on IOS.

**Comment by tongyy** [comment]
Did you fix the problem that didn't show up on the Android? you are facing another issue that it cannot be dragged?

**Comment by tonvlad88** [comment]
No, I was not able to fixed it. What I did was I put the Draggable component in my sidebar and it was showing but when the sidebar is show, it will be moved to the right. Now, it can be shown and displaying above my main component, but when I tried to drag it, it stucked, but on IOS it can be moved... Can this package be dragged over a view on android? Again, im using this Draggable component same as the assistive touch functionality  on IOS which can be used on all views.
